### PR TITLE
Form error messages to use localisation

### PR DESCRIPTION
**Pull request by knasher**

### CODE REVIEW

File: src/TransformCore/Bundle/AppBundle/Resources/translations/validators.en.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        type: "Please enter only letters for your last name"
+    plainpassword:
+        length:
+            max: "Please make sure your password is at most {{ limit }} characters long"
+            min: "Please make sure your password is at least {{ limit }} characters long"
+        notblank: "Please enter your chosen password"
+        regex:
+            letter: "Please make sure your password contains at least 1 letter"
+            number: "Please make sure your password contains at least 1 number"
+            symbol: "Please make sure your password contains at least 1 symbol (e.g. !@#$%^*_-)"
```
Comment (eddiejaoude) [inline src/TransformCore/Bundle/AppBundle/Resources/translations/validators.en.yml]: Needs a blank line at the end of every file

**Comment by knasher** [comment]
Could someone review this one too?

File: composer.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
         "twbs/bootstrap": "3.3.*",
         "jquery/jquery":  "1.11.1",
         "friendsofsymfony/user-bundle": "~2.0@dev",
-        "transformcore/csr-fast-stream-domain-model": "dev-master",
+        "transformcore/csr-fast-stream-domain-model": "dev-feature/CSR-345",
```
Comment (eddiejaoude) [inline composer.json]: This will need to go back to `dev-master` once we merge in D/M

**Comment by eddiejaoude** [comment]
You need to update lock file as well

File: src/TransformCore/Bundle/AppBundle/Resources/translations/validators.en.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+error:
+    dateOfBirth:
```
Comment (eddiejaoude) [inline src/TransformCore/Bundle/AppBundle/Resources/translations/validators.en.yml]: The `yml` path should be the same as the original form.

Comment (knasher) [inline src/TransformCore/Bundle/AppBundle/Resources/translations/validators.en.yml]: Have logged this as Tech Debt https://github.com/TransformCore/CSR-Fast-Stream-frontend/issues/69

**Pull request merged**